docs/pages/pmd/languages/language_properties.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Language configuration
+permalink: pmd_languages_configuration.html
+author: Clément Fournier
+last_updated: July 2022 (7.0.0)
+tags: [languages]
+keywords: [pmd, cpd, options, command, auxclasspath, language, properties]
+summary: "Summary of language configuration options and properties"
+---
+
+# Language properties
+
+Since PMD 7.0.0, languages may be directly configured via properties.
+
+TODO describe CLI syntax
+TODO describe env var syntax
+
+As a convention, properties whose name start with an *x* are internal and may be
+removed or changed without notice.
+
+## Common language properties
+
+All languages support the following properties:
+- `suppressMarker`: A string to detect suppression comments. The default is `NOPMD`,
+so e.g. in Java, a comment `// NOPMD` will suppress warnings on the same line.
+
+## Java language properties
```

Comment:
I think it would be better not to split up this documentation and instead link to this page from the language-specific pages. This allows having the shared language properties close to the language-specific ones, along with the different ways to set these properties

Reply:
Yes, either way has advantages. Let's keep the language properties for now here on this single page and link from the language specific pages (I've updated #3175) to here. If this turns out to be not helpful, we can change it at any time. At the moment, we don't have many properties, so a single page is not too crowded yet.